### PR TITLE
[GPT-750][GPT-751]handlebars template unified

### PR DESCRIPTION
**Pull request by nickbalestra**

### CODE REVIEW

**Comment by nickbalestra** [comment]
Solve #16

**Comment by nickbalestra** [comment]
Few things could be modularized further, but I guess is good to get starting to use this and then slowly improve on incremental refactoring.

Mainly now the jade-template unified has been merged in this monorepo

File: packages/oc-template-handlebars-compiler/test/compileServer.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      }
+    }
+  },
+  componentPath,
+  publishPath,
+  publishFileName
+};
+
+test('Should correctly compile the server', done => {
+  // fs.ensureDirSync(publishPath);
```
Comment (matteofigus) [inline packages/oc-template-handlebars-compiler/test/compileServer.test.js]: what is this commented line? is it actually needed?

File: packages/oc-template-handlebars-compiler/test/compileView.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      }
+    }
+  },
+  componentPath,
+  publishPath,
+  publishFileName
+};
+
+test('Should correctly compile the view', done => {
+  // fs.ensureDirSync(publishPath);
```
Comment (matteofigus) [inline packages/oc-template-handlebars-compiler/test/compileView.test.js]: same here

File: packages/oc-template-jade-compiler/lib/compileView.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const format = require('stringformat');
```
Comment (matteofigus) [inline packages/oc-template-jade-compiler/lib/compileView.js]: why this? Looks like it's not needed

File: packages/oc-template-handlebars-compiler/lib/compile.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+const async = require('async');
+const compileServer = require('./compileServer');
+const compileStatics = require('./compileStatics');
+const compileView = require('./compileView');
+const fs = require('fs-extra');
+const getUnixUtcTimestamp = require('oc-get-unix-utc-timestamp');
+const path = require('path');
+
+// OPTIONS
+// =======
```
Comment (matteofigus) [inline packages/oc-template-handlebars-compiler/lib/compile.js]: Same as previous PR, can we make this list of options alphabetic and comprehensive? (ocPackage missing)

File: packages/oc-template-jade-compiler/lib/compileView.js (hunk 3 above)
Comment (nickbalestra) [inline packages/oc-template-jade-compiler/lib/compileView.js]: needed down -> https://github.com/opencomponents/base-templates/blob/04b0e91dac7966ec490fb21cf24f05d50a8a02fd/packages/oc-template-jade-compiler/lib/compileView.js#L50

**Comment by nickbalestra** [comment]
@matteofigus thanks for the inputs, they should be addressed

**Pull request merged**